package.json

```diff
@@ -22,9 +23,19 @@
   },
   "homepage": "https://tc39.es/",
   "engines": {
-    "node": ">=14.8"
+    "node": ">=14.8.0"
```

Comment:
this is fine but also unnecessary; `>=14.8` and `>=14.8.0` are identical.

Reply:
Known, but I used the exact version number here to help explain why there is so much code in here to support top-level await.